lib/platform/azure/index.ts

```diff
@@ -554,6 +554,8 @@ export async function ensureComment({
       'Comment is already update-to-date'
     );
   }
+
+  return true;
```

Comment:
`Promise.resolve(true);`?

Reply:
not strictly nessessary, because we have a `async` function